Frameworks/Foundation/NSArray.mm

```diff
@@ -996,21 +995,29 @@ - (NSIndexSet*)indexesOfObjectsAtIndexes:(NSIndexSet*)indexSet
 }
 
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
 - (NSArray*)initWithContentsOfURL:(NSURL*)aURL {
-    UNIMPLEMENTED();
-    return StubReturn();
+    NSData* data = [NSData dataWithContentsOfURL:aURL];
+    return static_cast<NSArray*>(
+        CFPropertyListCreateWithData(nullptr, static_cast<CFDataRef>(data), kCFPropertyListImmutable, nullptr, nullptr));
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Caveat
+ @Notes Only file:// URLs supported. atomically parameter not supported.
```

Comment:
I'm not sure about noting either of these here; they're gaps from NSData, really, and if we fix those this works "for free"